CHANGELOG.md

```diff
@@ -55,6 +55,8 @@ different versioning scheme, following the Haskell community's
 * Fixed a race condition when `bond::ext::gRPC::io_manager::shutdown` and
   `bond::ext::gRPC::io_manager::wait` are called concurrently.
 * Fixed a race condition during `bond::ext::gRPC::unary_call` destruction.
+* Added `bond::ProtobufBinaryWriter<Buffer>` for serialization to Protocol
+  Buffers binary format.
```

Comment:
Should we mention what version of PB are we writing to?

Reply:
If you mean `proto2` vs `proto3` then the encoding is compatible (same?). The difference is mostly about IDL and default behavior.